src/HelloServlet.java

```diff
@@ -0,0 +1,64 @@
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import java.util.*;
+
+@WebServlet("/HelloServlet")
+
+
+public class HelloServlet extends HttpServlet {
+
+
+    private String globalVar = null;
+
+    protected void doGet(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        response.setContentType("text/html");
+        PrintWriter writer = response.getWriter();
+        writer.println("<p>I`m from GET!</p>");
+        writer.close();
+        System.out.println("GET globalVar is: " + globalVar);
+
+
+    }
+
+    protected void doPost(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        response.setContentType("text/html");
+        PrintWriter writer = response.getWriter();
```

Comment:
Initialization of `PrintWriter` should be in block `try{ }`. For more information about `try(){ ... }` 
on: https://www.baeldung.com/java-try-with-resources